sklearn/cluster/tests/test_optics.py

```diff
@@ -217,236 +216,168 @@ def test_cluster_sigmin_pruning(reach, n_child, members):
-         0.651366, 0.633434, 0.459580, 0.859221, 0.552291, 0.591751, 0.819965,
-         0.669977, 1.185083, 0.499338, 0.533907, 0.752871, 0.571388, 0.539772,
-         0.449182, 1.025370, 0.365182, 1.321022, 0.926213, 0.886360, 0.562272,
-         0.669977, 0.796046, 0.557598, 0.596776, 0.672336, 0.659111, 0.453719,
-         0.477716, 0.477716, 1.592069, 0.591751, 0.539772, 0.641612, 0.946254,
-         0.744165, 0.386679, 0.593825, 0.539772, 0.449182, 0.604273, 0.794951,
-         0.752871, 0.539772, 0.648732, 0.469030, 0.665088, 1.332700, 1.341388,
-         0.533507, 0.544212, 1.025992, 0.645967, 0.612945, 0.868492, 0.648732,
-         0.752300, 0.624444, 1.219748, 0.446161, 0.520818, 0.469044, 0.669977,
-         0.926213, 0.638752, 0.762191, 0.922273, 0.794951, 0.606885, 0.669977,
-         0.550113, 0.641067, 0.733026, 0.604273, 0.648732, 0.533507, 0.746506,
-         0.733026, 0.980683, 0.538209, 0.669977, 0.469030, 0.648732, 0.609190,
-         1.219748, 0.373113, 0.539772, 1.744047, 1.004716, 0.926213, 0.562272,
-         0.752871, 0.538656, 0.449182, 0.365182, 0.469030, 0.446161, 0.484404,
-         0.768592, 0.648732, 0.655156, 0.521651, 0.779432, 0.446161, 0.596776,
-         0.538209, 0.726740, 0.539772, 0.469030, 0.521651, 0.561950, 0.601851,
-         0.533907, 0.922273, 1.248265, 0.476800, 0.737990, 0.817970, 0.792127,
-         0.533907, 0.486038, 0.624444, 0.798241, 0.476800, 1.059373, 0.645967,
-         0.619940, 0.528726, 0.669977, 0.865406, 0.980683, 0.980683, 0.834671,
-         1.001353, 0.752871, 0.449182, 1.096520, 0.449182, 0.593825, 0.636558,
-         0.762191, 0.638591, 0.538209, 0.865406, 0.779432, 0.469044, 0.645967,
-         0.557598, 0.499338, 0.484404, 0.515686, 0.794951, 0.619456, 0.733026,
-         0.821769, 0.752300, 0.643302, 0.636558, 0.655156, 0.655156, 0.484404,
-         0.648732, 0.726023, 0.365182, 0.606885, 0.499338, 0.520818, 0.612945,
-         0.446161, 0.557598, 0.469044, 1.134650, 0.629401, 0.538656, 0.561950,
-         1.364861, 0.459580, 1.025370, 0.980304, 0.607592, 0.533907, 1.134650,
-         0.446161, 0.629962]
-
-    # FIXME: known failure in 32bit Linux; numerical imprecision results in
-    # different ordering in quick_scan
-    if _IS_32BIT:  # pragma: no cover
-        assert_allclose(clust.reachability_, np.array(v), rtol=1e-2)
-    else:
-        # we compare to truncated decimals, so use atol
-        assert_allclose(clust.reachability_, np.array(v), atol=1e-5)
+    # Tests against known extraction array
+    # Does NOT work with "euclidean", because sklearn euclidean has worse
+    # numerics
+    clust = OPTICS(min_samples=5, metric='minkowski').fit(X)
+
+    # we compare to truncated decimals, so use atol
+    assert_array_equal(clust.ordering_, np.array(o))
+    assert_array_equal(clust.predecessor_[clust.ordering_], np.array(p))
+    assert_allclose(clust.reachability_[clust.ordering_],
+                    np.array(r), atol=1e-15)
```

Comment:
I guess we don't need such a high precision here? I'm not an expert on these things but I think there might be some unexpected numeric issues in other environments (we only test some environments here). Maybe using the default value provided by ``assert_allclose``?